test/parallel/test-stream-readable-readingMore.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+const Readable = require('stream').Readable,
+	assert = require('assert');
```

Comment:
Please use a new `const` statement on each line (drop the comma separated declarations).